src/pages/Room/RoomDetailPage/hooks/useDetailMap.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+interface MapProps {
+  latitude: number | undefined
+  longitude: number | undefined
+}
+function useDetailMap({ latitude, longitude }: MapProps) {
+  const [map, setMap] = useState<naver.maps.Map | null>(null);
+
+  useEffect(() => {
+    if (!map && latitude && longitude) {
+      const newMaps = new naver.maps.Map('map', {
+        center: new naver.maps.LatLng(latitude, longitude),
+        maxZoom: 18,
+        minZoom: 14,
+        logoControl: false,
+        zoomControl: true,
+        scrollWheel: false,
+        draggable: true,
+        zoomControlOptions: {
+          position: naver.maps.Position.TOP_LEFT,
+        },
+      });
+      setMap(newMaps);
```

Comment:
```suggestion
      setMap(newMaps);
      return () => {
        newMaps.destory();
      }
```
map 객체를 파괴하는 절차가 필요합니다.

Reply:
수정했습니다. [da21295](https://github.com/BCSDLab/KOIN_WEB_RECODE/pull/39/commits/da21295c12e52151b5010b7031bd2f2d42d0505f)